docs/patterns/messaging-bridge-content.md

```diff
@@ -0,0 +1,90 @@
+The Messaging Bridge design pattern is a way to integrate disparate systems built on top of different messaging infrastructures.
+
+## Context and problem
+
+Many organizations end up in a situation in which their portfolio of IT systems uses multiple messaging infrastructures, e.g. by way of mergers and acquisitions. Others try to extend their current on-premises systems with cloud-hosted components for ease of maintenance, cost-effectiveness or other reasons. Yet others find themselves in a position where they need to integrate their newer systems with legacy ones. 
+
+One common solution to these challenges is dedicated integration through HTTP-based web services. In that approach systems being integrated are modified to include HTTP endpoints and/or clients in order to exchange the data.
+
+This approach has a number of drawbacks:
+- the applications being integrated must be modified (by adding an HTTP client on one side and an HTTP request handler on the other)
```

Comment:
```suggestion
- the systems must be modified (by adding an HTTP client on one side and an HTTP request handler on the other), retested, and redeployed.
```

In the earlier text, we used the term "system" so we should be consistent with that.